charts/refinery/values.yaml

```diff
@@ -255,6 +255,13 @@ tolerations: []
 
 affinity: {}
 
+podDisruptionBudget:
+  enabled: false
+  labels: {}
+  annotations: {}
+  minAvailable:
```

Comment:
Lets comment this out with a note that the field is allowed, but not at the same time as `maxUnavailable`.  Then lets add a check to `NOTES.txt` that both are not set at the same time.

Reply:
Added comment

Reply:
@gjacquet please also comment out `minAvailable` in the vaules.yaml and add a new section in `NOTES`.txt that will fail the install if both `minAvailable` and `maxUnavailable` are set.  I believe this is necessary because `PodDisruptionBudget` doesn't allow both being set at the same time.

Reply:
Done